src/hb-aat-layout-trak-table.hh

```diff
@@ -58,6 +65,13 @@ struct TrackTableEntry
 
 struct TrackData
 {
+  inline bool apply (hb_aat_apply_context_t *c) const
```

Comment:
We probably don't need apply here or in TrackTableEntry. Basically just add them when you hook them up.

Reply:
Makes sense. Thanks